atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresVersionCheckTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.dbkvs.impl.postgres;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.ArgumentMatcher;
+import org.mockito.Mockito;
+import org.slf4j.Logger;
+
+public class PostgresVersionCheckTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void shouldLogErrorOn_9_1_24() {
+        Logger log = Mockito.mock(Logger.class);
+        PostgresVersionCheck.checkDatabaseVersion("9.1.24", log);
+        Mockito.verify(log).error(Mockito.argThat(new ArgumentMatcher<String>() {
+            @Override
+            public boolean matches(Object arg) {
+                return ((String)arg).contains("The minimum supported version is {}");
```

Comment:
This line probably breaks checkstyle and findbugs. Please run locally (`atlasdb-dbkvs-tests:checkstyleTest` and `atlasdb-dbkvs-tests:findbugsTest`) and check the reports.